src/routes/profile/(components)/TournamentBlock.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { User, Zap, Star } from 'lucide-svelte';
+
+  export let title = '';
```

Comment:
It's better to not provide the value so that the linter will punch you if you forget one of the fields